sdk/storage/Azure.Storage.Blobs.ChangeFeed/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.0.0-preview.17 (Unreleased)
+## 12.0.0-preview.17 (2021-11-02)
 - Added support for service version 2020-12-06.
 - This release contains bug fixes to improve quality.
 - Fixed bug where Segment.GetCursor() would throw an ArgmentOutOfRangeException if the Segment has no Shards.
```

Comment:
I know this isnt apart of your PR, but ArgmentOutOfRangeException get corrected to ArgumentOutOfRangeException?

Reply:
Yes I can do that